### PR TITLE
Adding a new folders from workshop

### PR DESCRIPTION
**Pull request by knabben**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
This is the first PR for the remote workshop.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: brazil/learners-2022/knabben.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# KNABBEN
+
+Hello!
```
Comment (rikatz) [inline brazil/learners-2022/knabben.md:3]: nit: please add one more "!" :P

**Review by rikatz — CHANGES_REQUESTED** [review]
Please add more characters

**Comment by knabben** [comment]
> Please add more characters

Added a new line

**Pull request merged**